cloudsync_onedrive.py

```diff
@@ -554,7 +554,7 @@ def list_ns(self, recursive: bool = True, parent: Namespace = None) -> List[Name
 
     @memoize
     def _check_ns(self, nsid, conn_id_for_memo):                                 # pylint: disable=unused-argument
-        res = self._direct_api("get", "/drives/%s/items/%s" % (nsid, self.namespace.api_root_oid), raw_response=True)
+        res = self._direct_api("get", f"/drives/{nsid}/items/{self.namespace.api_root_oid}")
```

Comment:
was ` raw_response=True` deleted on purpose?

looks like this is causing some test failures.

Reply:
Accidental. There were a couple other typos as well